aea/cli/scaffold.py

```diff
@@ -360,3 +366,47 @@ def _scaffold_non_package_item(
     except Exception as e:
         os.remove(dest)
         raise click.ClickException(str(e))
+
+
+def add_contract_abi(ctx: Context, contract_name: str, contract_abi_path: str) -> None:
+    """
+    Add the contract ABI to a contract scaffold.
+
+    :param ctx: the CLI context.
+    :param contract_name: the contract name.
+    :param contract_abi_path: the contract ABI path.
+    """
+
+    # Get some data from the context
+    to_local_registry = ctx.config.get("to_local_registry")
+    author_name = ctx.agent_config.author
+
+    # Create the build directory and copy the ABI file
+    src = Path(contract_abi_path)
+
+    contract_dir_root = (
+        Path(str(ctx.agent_config.directory)) if to_local_registry else Path()
+    )
+    contract_dir = contract_dir_root / Path("contracts") / Path(contract_name)
```

Comment:
```suggestion
    contract_dir = contract_dir_root / Path("contracts") / Path(contract_name)
```
get constant for `contracts` from aea.configuration